__init__.py

```diff
@@ -49,7 +49,7 @@ def upload_and_create_url(self, log_str):
             f.write(log_str)
         os.close(fd)
         cmd = 'cat ' + path + ' | nc ' + self.host + ' 9999'
-        return check_output(cmd, shell=True).strip('\n\x00')
+        return check_output(cmd, shell=True).decode().strip('\n\x00')
```

Comment:
Should we reorder it to `.strip(b'\n\x00').decode()` in case the `\x00` character isn't properly represented in the locale or is that standard?

Reply:
decode defaults to utf-8 and \n and \x00 is in the ascii range which are represented unmodified if I've understood things.

For expandability it might be good to reorder though...

Reply:
Should be good how it is then. LGTM, merging.